bin/rollup.config.js

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import path from 'path';
+import resolve from '@rollup/plugin-node-resolve';
+
+export default {
```

Comment:
This looks great. Just one request. Can we rename this rollup config to `rollup.migrate.config.js` or `migrate.rollup.config.js`? It's likely that we'll need more than one of these in the future.

Reply:
Sure!